src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+  
+  &__image {
+    height: 134px;
+    margin: 0 3px;
+  }
+
+  &__name {
+    margin-top: 40px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-dark;
+    transition-duration: 30ms;
```

Comment:
Are you sure that it should be so fast? 
Let's create and use a variable for transition duration to make it the same per page